### ISSUE TITLE
Acceptance test 2020-10-14

### ISSUE THREAD

**Issue by pintergreg** [SzFMV2020-Osz/AutomatedCar-B#106]
@SzFMV2020-Osz/company Elfogadván a #103 és #105 PR-eket a következőt tudom mondani a szoftverről:

Nagyon szépen megvan a pálya rajzolása a poligonokkal együtt, 

- [ ] kivéve a T elágazást. Itt nem tudom, hogy csak nem került feldolgozásra vagy eleve nem is volt készen poligon hozzá.

Ez #4 és #2 jelentős részét lefedi.

- [ ] Nincs viszont észlelhető hajtáslánc, továbbra is a skeleton-ből származó mozgás van.
- [ ] A space billentyű lenyomása elront mindent
    - odakommenteltem a PR-ben, hogy nem értem mire van, de minden protokollt és előírást felrúgva ennek ellenére elfogadtam. (Ha ez valami éles szitu lenne akkor sokba fájna ez a cégnek.)
- [ ] A HMI-ből nem látszik még semmi, kivéve, hogy két billentyűt is kezel
    - [ ] a debug kapcsoló sem aktív még amiről szó volt, ennek következménye, hogy a szenzorháromszög nem demózható.

Ezeket ilyen acceptance-test jelleggel írom, mintha a demón vizsgálnánk. Egyébként a két kódbázis összeboronálása igen szép munka és ahogy kivettem igen sok időtök is ment rá. Minden elismerésem.

**Comment by aaronrancsik** [comment]
> kivéve a T elágazást. Itt nem tudom, hogy csak nem került feldolgozásra vagy eleve nem is volt készen poligon hozzá.

Jelenleg azok az elemek amiknek több mint három polygonja van, azokra kell külön külön polygonszám szerint avalonias szűrő (olyan ami már van út elemekre is) illetve az osztályt is létre kell ezekhez hozni, ez valószínűleg nem fog első pót sprintre bekerülni, de majd meglátjuk. Jelenleg ezeknél is csak az első három polygon jelenik meg.


>  A space billentyű lenyomása elront mindent

Utána járok 


> nem tudom miért van szellemautónk. :)

Ezt mindjart javítom is ha még valaki nem tette volna meg

**Comment by aaronrancsik** [comment]
@pintergreg A space mit ront el pontosan? nálam úgy működik ahogy azt gondoltuk. 
Ha nyomva tartjuk a space-t akkor a nyilakkal történő mozgás felgyorsul, de az csak debug célra van. PowerTrain esetén ilyen úgyse lesz. Ez majd a  négyzet km-es pályánál volt kényelmes, de arra még sokat kell optimalizálni mert a jelenlegi kód azt nem bírja....

**Comment by pintergreg** [comment]
Eltűnt az autó és ugrott egyet a képernyő. Teljesen jogos, hogy ezt oda kellett volna írnom.

**Comment by pintergreg** [comment]
Értem már mit csinál.
Most csak úgy sikerült reprodukálnom, hogy a pálya elején „hátramenetben” nyomom meg, akkor kiugrik a pályáról és a kamera nem követi.

![Képernyőkép_2020-10-14_15-50-34](https://user-images.githubusercontent.com/3854784/95998292-04b5a200-0e35-11eb-9e3f-1ab6ca63dd60.png)

**Comment by aaronrancsik** [comment]
@pintergreg  Nem tudom hova írjam, de majd 19 óra magasságában lesz egy pr ami javítja ezt:

> kivéve a T elágazást. Itt nem tudom, hogy csak nem került feldolgozásra vagy eleve nem is volt készen poligon hozzá.

Van arra esély h, az még akkor bekerüljön majd mster-re? és akkor ezt is tudjuk holnap demozni

**Comment by pintergreg** [comment]
Igen, a más nem holnap reggel. Legyen rajta Approve csapattárstól azért.

**Comment by zsofiath** [comment]
A T elágazással van egy olyan gond hogy több poligonja van, mint a többi útnak. Ezt próbáltam ciklussal kezelni itt https://github.com/SzFMV2020-Osz/AutomatedCar-B/pull/107 . Egyelőre nem működik. Jelenleg én úgy látom hogy az lesz a megoldás, hogy ennek is lesz egy külön típusa oszt beégetem fixen a poligon számot... piszok béna, de egyszerűen nem tudok jobbat.

**Comment by aaronrancsik** [comment]
>  Jelenleg én úgy látom hogy az lesz a megoldás, hogy ennek is lesz egy külön típusa oszt beégetem fixen a poligon számot... piszok béna, de egyszerűen nem tudok jobbat.

Dáviddal ezt megcsináljuk mindjárt 17-19 ig meg lesz terveink szerint.
